### PR TITLE
update document-input page by adding suggestions

### PR DESCRIPTION
**Pull request by fakela**
update document-input page by adding suggestions from #249

### CODE REVIEW

File: apis-overview/document-inputs.md [hunk 1]
```diff
@@ -21,27 +21,27 @@ We currently support three different payload formats when sending your document
...
-
 
 ## Working with PDFs
 Our APIs support multi-page PDF files. However, the processing time may be longer compared to image processing, because we need to convert the PDF to image first.
 
 ### Tips for Working With PDFs
-- **Best with digital documents**: PDF documents should be in form of digital documents for maximum performance.
+- **Use native pdf**: Native PDFs are easier to process and has a higher performance and accuracy rate than scanned PDF.
 - **Working with several pages**:  Your PDF may contain several pages. In this scenario, you will receive both predictions at the page level and at the document level. See [JSON response scheme](https://developers.mindee.com/docs/prediction#json-responsep) documentation for more details.
+- **Using the cutPdf Library**: The cutPDF library is available when using the SDK. This library is used when sending multi pages pdf of more than 5 pages.
```
Comment (fharper) [inline apis-overview/document-inputs.md:46]: `cutPDF` isn't a librairy, it's an option in the SDKs

File: apis-overview/document-inputs.md [hunk 2]
```diff
@@ -21,27 +21,27 @@ We currently support three different payload formats when sending your document
...
 
 ### Supported Filetypes
-We currently support JPG, PNG and WEBP format. 
-> 📘 Info
+We currently support `.jpg`,`.png` and `.webp` format. 
+
+> 📘 **Info**
 >
-> Always choose JPG over PDF for scans and photos (as images are processed faster).
+> Always choose images over PDF for scans and photos (as images are processed faster).
```
Comment (fharper) [inline apis-overview/document-inputs.md:31]: When possible, send images over...


Also, let's add something about the fact that the photos or PDF work just fine, they aren't just AS fast.

Comment (fharper) [reply apis-overview/document-inputs.md:31]: Actually, let's say something more like "If you want faster process time..."

**Review by fharper — CHANGES_REQUESTED** [review]

File: apis-overview/document-inputs.md (hunk 1 above)
Comment (fakela) [reply apis-overview/document-inputs.md:46]: I saw this library thing in the docs and thought it was a library

Comment (fharper) [reply apis-overview/document-inputs.md:46]: Where? We should fix it if it's not clear.

Comment (fakela) [reply apis-overview/document-inputs.md]: https://developers.mindee.com/docs/nodejs-invoice-ocr

**Review by fharper — APPROVED** [review]
LGTM

**Pull request merged**